crypto/.classpath

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+  <classpathentry kind="src" path="src/main/java" including="**/*.java"/>
```

Comment:
src/main/java -> src/

Reply:
pls refactor your source code structure, we need not a java subfolder under our java-common project.

src/main/java/com/arxanfintech/common/util/Utils.java -> src/com/arxanfintech/common/util/Utils.java